<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SideBarTab</class>
 <widget class="QWidget" name="SideBarTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>106</width>
    <height>32</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>32</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>32</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="styleSheet">
   <string notr="true">/*ColorTokenIcon;Button;lClose;normal;off;colorToken.icon-primary*/

SideBarTab
{
border-radius:6px;
}

SideBarTab[selected=true]
{
background-color: #FFFFFF; /*colorToken.page-background*/
}

SideBarTab[selected=false]
{
background-color:  transparent;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QToolButton" name="lIcon">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Lato</family>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="class" stdset="0">
      <string notr="true">icon-label</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lTitle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lCounter">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="lClose">
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="icon">
      <iconset resource="../Resources_common.qrc">
       <normaloff>:/x_solid_small</normaloff>:/x_solid_small</iconset>
     </property>
     <property name="class" stdset="0">
      <string notr="true">icon-label</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources_common.qrc"/>
 </resources>
 <connections/>
</ui>
